src/community/roadmap/index.md

```diff
@@ -40,9 +41,6 @@ We're working to:
 We're getting ready to:
```

Comment:
Any upcoming stuff we want to add here

Reply:
It'd be great to mention the guidance on adding findings from research that @christopherthomasdesign is looking at.

Reply:
Think we should mention that we're planning the next priorities too.

Reply:
Have added these in a commit!

Reply:
Looks good, thanks for adding and approving!